### PR TITLE
tested and created the GET /api endpoint 

### PR DESCRIPTION
**Pull request by Garyismatic**
to retrieve an object containing details about the other available endpoints

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -4,10 +4,23 @@ const db = require('../db/connection')
...
+        it('responds with an object containing a list of the available endpoints', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+            .then(({body}) => {
+                expect(body).toEqual({endpoints})
+            })
+        })
+    })
+})
```
Comment (nc-niamh-smith) [inline __tests__/api.test.js:23]: I think another test checking the keys are the ones you expect :)

**Review by nc-niamh-smith — COMMENTED** [review]
Another test might be necessary here, but great so far, it's very clear and simple!

File: __tests__/api.test.js (hunk 1 above)
Comment (Garyismatic) [reply __tests__/api.test.js:23]: I'm not sure how to do this and to get it to fail, and if I manually type it in each time I update the file I would need to update the test?

Comment (Garyismatic) [reply __tests__/api.test.js:23]: I have pushed up a test to my main branch for it, if you have the time to check it out

**Pull request merged**